packages/core/chrome/core-chrome-browser-internal/src/chrome_service.test.tsx

```diff
@@ -202,10 +202,16 @@ describe('start', () => {
     });
 
     it('renders the default project side navigation', async () => {
-      const { chrome } = await start();
+      const startDeps = defaultStartDeps([new FakeApp('DefaultNavApp')]);
+      const { navigateToApp } = startDeps.application;
+      const { chrome, service } = await start({ startDeps });
 
       chrome.setChromeStyle('project');
 
+      navigateToApp('DefaultNavApp');
```

Comment:
This test has strange failures: the visible state the component has access to is always `false` in the testing environment.